src/pages/ReimbursementAccount/NonUSD/BusinessInfo/subSteps/BusinessType.tsx

```diff
@@ -5,24 +5,32 @@ import useOnyx from '@hooks/useOnyx';
 import useReimbursementAccountStepFormSubmit from '@hooks/useReimbursementAccountStepFormSubmit';
 import type {SubStepProps} from '@hooks/useSubStep/types';
 import getListOptionsFromCorpayPicklist from '@pages/ReimbursementAccount/NonUSD/utils/getListOptionsFromCorpayPicklist';
+import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import INPUT_IDS from '@src/types/form/ReimbursementAccountForm';
 
 type BusinessTypeProps = SubStepProps;
 
-const {BUSINESS_CATEGORY, APPLICANT_TYPE_ID} = INPUT_IDS.ADDITIONAL_DATA.CORPAY;
-const STEP_FIELDS = [BUSINESS_CATEGORY, APPLICANT_TYPE_ID];
+const {COUNTRY} = INPUT_IDS.ADDITIONAL_DATA;
+const {BUSINESS_CATEGORY, APPLICANT_TYPE_ID, BUSINESS_TYPE_ID} = INPUT_IDS.ADDITIONAL_DATA.CORPAY;
+const STEP_FIELDS_WITHOUT_BUSINESS_TYPE = [BUSINESS_CATEGORY, APPLICANT_TYPE_ID];
+const STEP_FIELDS = [BUSINESS_CATEGORY, APPLICANT_TYPE_ID, BUSINESS_TYPE_ID];
 
 function BusinessType({onNext, isEditing, onMove}: BusinessTypeProps) {
     const {translate} = useLocalize();
     const [reimbursementAccount] = useOnyx(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {canBeMissing: false});
+    const [reimbursementAccountDraft] = useOnyx(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {canBeMissing: true});
     const [corpayOnboardingFields] = useOnyx(ONYXKEYS.CORPAY_ONBOARDING_FIELDS, {canBeMissing: false});
+    const country = reimbursementAccountDraft?.[COUNTRY] ?? reimbursementAccount?.achData?.[COUNTRY] ?? '';
+    const isBusinessTypeRequired = country !== CONST.COUNTRY.CA;
```

Comment:
Oh I thought we would require it for CA, AU, and European countries like the PR description?

> Added a new picklist to step 3 of the global reimbursements flow. We should only gather this field for GBP, EUR and AUD accounts.